docs/articles/documentation/recipes/extract-reusable-test-code/create-helpers.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: docs
+title: Create Helpers
+permalink: /documentation/recipes/extract-reusable-test-code/create-helpers.html
+---
+# Create Helpers
+
+We recommend that you follow the [page model](use-page-model.md) pattern to extract reusable test code. This pattern allows you to abstract out both page structure and test logic.
```

Comment:
For me it'w written like exporting helpers and using PageModel are two mutually exclusive alternatives. You can use both of them. I think we should start this section from the reason why it's useful to extract repeatable test logic into helpers and we can mention that in this case you still can use PageModel patter in the same time.

Reply:
I've missed changes in PageModel recipe, so that's ok for me now